model/src/main/java/org/projectnessie/model/ContentKey.java

```diff
@@ -36,7 +36,7 @@
  * <p>For URL encoding, embedded periods within a segment are replaced with zero byte values before
  * passing in a url string.
  */
-@Value.Immutable(lazyhash = true)
+@Value.Immutable
 @JsonSerialize(as = ImmutableContentKey.class)
 @JsonDeserialize(as = ImmutableContentKey.class)
 public abstract class ContentKey implements Serializable {
```

Comment:
you can remove `Serializable`  also from `ContentKey` if you want. Later when I really need it for GC, I will add it back.

Reply:
I think this is actually a bug in the Immutables lib. Also we want to eventually make all model classes Serializable, so IMO the more correct solution currently is to remove `lazyhash = true`